packages/table/src/table.tsx

```diff
@@ -450,6 +450,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
+    public resizeRowsBasedOnColumnHeight(columnIndex: number) {
```

Comment:
we should probably also expose a method for resizing columns since they will surely ask for that next. We already have some of this code anyway, right?